ray-operator/config/samples/ray_v1alpha1_rayservice.yaml

```diff
@@ -36,7 +36,7 @@ spec:
         rayActorOptions:
           numCpus: 0.1
   rayClusterConfig:
-    rayVersion: '1.12.1' # should match the Ray version in the image of the containers
+    rayVersion: 'nightly' # should match the Ray version in the image of the containers
```

Comment:
I think we've established this, but we'll want to remove "nightly" after the 2.0.0 release or even once we have a release candidate.

Reply:
right. Now we only works with nightly. Later we can change it to 2.0.0